Allow Cesium3DTileStyle with conditionally undefined scaleByDistance, translucencyByDistance and distanceDisplayCondition

ShapovalovKL opened the pull request:
Allow to use StyleExpression for scaleByDistance. translucencyByDistance, distanceDisplayCondition which returns undefined for some objects

cesium-concierge commented:
Thank you so much for the pull request @ShapovalovKL! I noticed this is your first pull request and I wanted to say welcome to the Cesium community!

The [Pull Request Guidelines](https://github.com/CesiumGS/cesium/blob/main/CONTRIBUTING.md#pull-request-guidelines) is a handy reference for making sure your PR gets accepted quickly, so make sure to skim that.

* :x: Missing `CONTRIBUTORS.md` entry.
    * Please add yourself to the [contributors](https://github.com/ShapovalovKL/cesium/blob/main/CONTRIBUTORS.md) file!
* :x: Missing CLA.
    * Please send in a [Contributor License Agreement](https://github.com/CesiumGS/cesium/blob/main/CONTRIBUTING.md#contributor-license-agreement-cla) (CLA) and comment back here to let us know to check this!
* :grey_question: [CHANGES.md](https://github.com/CesiumGS/cesium/blob/main/CHANGES.md) was not updated.
    * If this change updates the public API in any way, please add a bullet point to `CHANGES.md`.
* :grey_question: Unit tests were not updated.
    * Make sure you've [updated tests](https://github.com/CesiumGS/cesium/tree/main/Documentation/Contributors/TestingGuide) to reflect your changes, added tests for any new code, and ran the code coverage tool.

Reviewers, don't forget to make sure that:

- [ ] Cesium Viewer works.
- [ ] Works in 2D/CV.

ShapovalovKL commented:
I signed the CLA and added myself to CONTRIBUTORS.md

ggetz commented:
Thanks for the PR @ShapovalovKL! Could you please:

* Add a unit test to `Cesium3DTileStyleSpec` to verify the `undefined` condition for each property
* Update `CHANGES.md` with a summary of the change

ggetz commented:
Thanks @ShapovalovKL!

To get CI to pass, please run `npm run prettier` to format all files and commit the result.

ShapovalovKL commented:
Updated CHANGES.md and added test to Vector3DTilePointsSpec. No need to add tests to Cesium3DTileStyleSpec, since it only checks getters and setters for properties and already checks getters and setters with StyleExpression

ggetz commented:
Thanks @ShapovalovKL for your first Cesium contribution!
